tests/VObject/Recur/EventIterator/HandleRDateExpandTest.php

```diff
@@ -54,7 +54,125 @@ function testExpand() {
 
         $result = array_map(function($ev) {return $ev->dtstart->getDateTime();}, $result);
         $this->assertEquals($expected, $result);
-    
+
+    }
+
+
+    function testExpandRDateWithoutTime() {
+        $input = <<<ICS
+BEGIN:VCALENDAR
+VERSION:2.0
+BEGIN:VEVENT
+UID:BDF720338B3EC8D2C09CE95CFF85C0B1-A26036CE0B219773
+DTSTAMP;VALUE=DATE-TIME:20151211T121402Z
+CREATED;VALUE=DATE-TIME:20151013T061627Z
+LAST-MODIFIED;VALUE=DATE-TIME:20151014T093742Z
+DTSTART;VALUE=DATE-TIME;TZID=Europe/Berlin:20151028T124500
+DTEND;VALUE=DATE-TIME;TZID=Europe/Berlin:20151028T141500
+SUMMARY:test summary
+DESCRIPTION:test desc
+SEQUENCE:1
+RDATE;VALUE=DATE:20151028,20151111,20151125,20151209,20160113,20160120,2016
```

Comment:
What's interesting with this example, is that I don't believe that we are _supposed_ to be able to support this particular syntax. As far as I know the value-type of `RDATE` needs to be identical to the value type of `DTSTART`. So if `DTSTART` is a `DATE-TIME`, then so must `RDATE` be...

Did you find this in the wild somewhere?